src/btree/bt_cursor.c

```diff
@@ -1964,6 +1964,16 @@ __wt_btcur_range_truncate(WT_CURSOR_BTREE *start, WT_CURSOR_BTREE *stop)
 
     WT_STAT_DATA_INCR(session, cursor_truncate);
 
+    /*
+     * All historical versions must be removed when a key is updated with no timestamp, but that
+     * isn't possible in fast truncate operations. Disallow fast truncate in transactions configured
+     * to commit without a timestamp (excluding logged objects as timestamps will be ignored).
```

Comment:
"will be ignored" could be replaced with "cannot be relevant" or "cannot be used"?

Reply:
Fixed in 6658175.